spec/requests/articles_controller_spec.rb

```diff
@@ -0,0 +1,133 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ArticlesController, type: :controller do
+  context 'articles user not authenticated' do
+    describe '#index' do
+      before { get :index }
+
+      it { should respond_with 200 }
+      it { should render_template 'index' }
+    end
+
+    describe '#show' do
+      subject { create :article }
```

Comment:
This should be a `before` block. The subject should be `get :index`.

Reply:
Done!